blaze/compute/sql.py

```diff
@@ -1181,7 +1181,11 @@ def compute_up(expr, data, **kwargs):
 
 
 @dispatch(Expr, ClauseElement)
-def post_compute(_, s, **kwargs):
+def post_compute(expr, s, **kwargs):
+    if isscalar(expr.dshape):
+        return select(s).scalar()
+    elif iscollection(expr.dshape) or istabular(expr.dshape):
+        return select(s).execute().fetchall()
```

Comment:
Similarly, we should do:

```python
return odo.into(s, pd.DataFrame)
```

The `fetchall()` will return a nested Python datastructure, not a Pandas object.

It'd be even better if we could detect when the SQLA object is a single column and `odo` it into a `pd.Series`